bugbug/models/bugtype.py

```diff
@@ -102,6 +141,32 @@ def get_labels(self):
             for keyword in bug_data["keywords"]:
                 if keyword in KEYWORD_DICT:
                     target[KEYWORD_LIST.index(KEYWORD_DICT[keyword])] = 1
+            whiteboard_data = bug_data["whiteboard"]
+            for whiteboard in whiteboard_data.split("["):
+                if whiteboard.startswith("rca - "):
+                    # Cases of type rca - X - Y
+                    rca_whiteboards = (
+                        re.sub("]", "", whiteboard[6:]).lower().split(" - ")
```

Comment:
whiteboard[6:] is a bit brittle, maybe better to split by "-" so we also support cases like `rca-`

Reply:
I couldn't find any cases with `rca-`, all of them had `rca - `.

Reply:
On second thoughts, I think it is possible someone adds `rca-` , since there is already a lot of irregularity.